app/src/selfRelease/java/org/radarcns/detail/DownloadProgress.kt

```diff
@@ -0,0 +1,130 @@
+package org.radarcns.detail
+
+import android.annotation.SuppressLint
+import android.content.Context
+import okhttp3.*
+import okhttp3.Interceptor
+import okio.*
+import org.slf4j.LoggerFactory
+import java.io.File
+import java.io.IOException
+import java.lang.Exception
+
+class DownloadProgress(context: Context, delegate: TaskDelegate) {
+    private val mContext: Context = context
+    private val mDelegate: TaskDelegate = delegate
```

Comment:
You can combine fields with arguments as such:
```suggestion
class DownloadProgress(
    private val mContext: Context,
    private val mDelegate: TaskDelegate,
) {
```